cli/src/commands/git/push.rs

```diff
@@ -766,15 +769,16 @@ fn find_bookmarks_targeted_by_revisions<'a>(
 ) -> Result<Vec<(&'a str, LocalAndRemoteRef<'a>)>, CommandError> {
     let mut revision_commit_ids = HashSet::new();
     if use_default_revset {
-        // remote_bookmarks(remote=<remote>)..@
-        let workspace_id = workspace_command.workspace_id();
-        let expression = RevsetExpression::remote_bookmarks(
-            StringPattern::everything(),
-            StringPattern::exact(remote_name),
-            None,
-        )
-        .range(&RevsetExpression::working_copy(workspace_id.clone()))
-        .intersection(&RevsetExpression::bookmarks(StringPattern::everything()));
+        let revs = workspace_command.settings().get_string("revsets.push")?;
+        // This obviously isn't correct
+        let mut map = TemplateAliasesMap::new();
+        map.insert("remote", remote_name);
+        let template = parse(&revs, &map)?;
+
```

Comment:
At this point its no longer clear to me how to proceed, as trying to mutate the existing `aliases_map` in the workspace_command_helper via `borrow_mut()` just doesn't work.